sys/clif/clif.c

```diff
@@ -323,7 +327,21 @@ ssize_t clif_get_attr(const char *input, size_t input_len, clif_attr_t *attr)
         attr->key_len = pos - attr->key;
     }
 
+    /* either the value is unquoted (0) or quoted (2) */
+    if (quotes != 0 && quotes != 2) {
+        DEBUG("Incorrect number of unescaped quotes found: %d\n", quotes);
+        goto error;
+    }
+
+    if (scan_value && !attr->value_len) {
+        DEBUG("Empty value found");
+        goto error;
+    }
```

Comment:
As
```suggestion
    if (!attr->value_len) {
        DEBUG("Empty value found");
        goto error;
    }
```
This could come right [after the while loop in the `if (scan_value)` block and save an expression and && operation](https://github.com/RIOT-OS/RIOT/pull/16148/files#diff-22876fad107bdc91cf9f4f9ac6887cdfd9d4e2a38c73580d357f5e8a12ce7495R322)

Reply:
will change

Reply:
See b38f23b9b9b0418376956da0e3d5ae60a65d1dae